tensorflow/tensorboard/components/tf-event-dashboard/tf-event-dashboard.html

```diff
@@ -194,6 +201,17 @@
           chartScaffold.chart().redraw();
         }
       },
+
+      toggleYScale: function(e) {
+        var b = this.parentNode.querySelector('.log-option-button');
+        var c = this.parentNode.querySelector('vz-line-chart');
```

Comment:
As currently written, clicking any log/linear button always modifies the first chart. Here is a fix:
```
      toggleYScale: function(e) {
        var currentTarget = Polymer.dom(e.currentTarget);

        var b = currentTarget.parentNode.querySelector('.log-option-button');
        var c = currentTarget.parentNode.querySelector('vz-line-chart');
        if (c !== null) {
          c.yScale = c.yScale === 'log' ? 'linear' : 'log';
          b.icon = c.yScale === 'log' ? 'line-weight' : 'reorder';
          c.redraw();
        }
      },
```